src/utils.c

```diff
@@ -85,3 +87,41 @@ bool remove_ending_wildcard(char *elem) {
 bool strstarts(const char *str, const char *prefix) {
     return strncmp(str, prefix, strlen(prefix)) == 0;
 }
+
+bool is_table_range_var_in_list_of_tables_string(
+    const RangeVar *table_range_var, const char *list_of_tables_str) {
+    Oid target_table_id =
+        RangeVarGetRelid(table_range_var, AccessExclusiveLock, false);
+    List *allowed_policies_list = NULL;
+    ListCell *table_cell = NULL;
+
+    if (!SplitIdentifierString(pstrdup(list_of_tables_str), ',',
+                               &allowed_policies_list)) {
+        return false;
+    }
+
+    foreach (table_cell, allowed_policies_list) {
+        List *qual_name_list;
```

Comment:
Do we need to free this list?

Reply:
The list is consumed by `makeRangeVarFromNameList`. I also thought the `RangeVar` should be freed, but it's not done in Postgres upstream codebase so I left it out.

Freeing is generally not a necessity with Postgres libs because it uses [arena allocation](https://www.cybertec-postgresql.com/en/memory-context-for-postgresql-memory-management/) - all allocations are freed at the end of the context.